src/Razor/test/Microsoft.AspNetCore.Razor.Language.Test/TestFiles/IntegrationTests/ComponentRuntimeCodeGenerationTest/ScriptTag_WithErrorSuppressed/TestComponent.codegen.cs

```diff
@@ -15,13 +15,13 @@ protected override void BuildRenderTree(Microsoft.AspNetCore.Components.RenderTr
         {
             base.BuildRenderTree(builder);
             builder.OpenElement(0, "div");
-            builder.AddContent(1, "\r\n    ");
+            builder.AddMarkupContent(1, "\r\n    ");
             builder.OpenElement(2, "script");
             builder.AddAttribute(3, "src", "some/url.js");
             builder.AddAttribute(4, "anotherattribute", true);
             builder.AddContent(5, "\r\n        some text\r\n        some more text\r\n    ");
```

Comment:
An example of newlines still making it through.

Reply:
Yup. Looks like `ComponentMarkupBlockPass` won't rewrite if the tag has attributes, which means we call `AddContent` on the entire content. To fix this, we'll have to switch everything to be `AddMarkupContent` but I'm not sure if the perf tradeoff is worth it.

Reply:
Explain? Is this a bug in `ComponentMarkupPass`?

Reply:
Looked at the code more. It wasn't because of attributes but this was because it was `script` tag. We special case it. There is no bug. There is still a case that will allow newlines to be encoded though,
```
@{
    var x = "a\r\nb";
}

<div>a @x b</div>
```

will call,
`AddContent(x)`

which will output,
`<div>a &#xD;&#xA; b`</div>